contrib/format-excel/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Please remove we have common logback-test.xml for all tests.

Reply:
Removed.